File: HelloWorld/results/HelloWorld.swift
class TestClass {
  func testMethod() -> Void {
      let value = 1 + 2 * 3 - 4
      let map_: [String: Int]? = [
        "a": 5,
        "b": 6
      ]
      let text = "Hello world! value = \(value), map[a] = \(map_!["a"]!)"
      print(text)
  }
}

TestClass().testMethod()

File: HelloWorld/results/HelloWorld.py
class TestClass:
    def test_method(self):
        value = 1 + 2 * 3 - 4
        map_ = {
          "a": 5,
          "b": 6,
        }
        text = "Hello world! value = %s, map[a] = %s" % (value, map_["a"], )
        print text

try:
    TestClass().test_method()
except Exception as err:
    print "Exception: " + err.message